config.example/group_vars/all.yml

```diff
@@ -67,3 +67,8 @@
 
 #software_remove_packages:
 #  - popularity-contest
+
+#Uncomment the following three lines to use the Mellanox RoCE high-speed backend
+#kube_network_plugin: flannel
+#kube_network_plugin_multus: true
+#roce_backend: true
```

Comment:
This variable needs to be uncommented by default or a more robust check done later, the QA build is failed:

```
fatal: [virtual-mgmt-0]: FAILED! => 

  msg: |-

    The conditional check 'roce_backend' failed. The error was: error while evaluating conditional (roce_backend): 'roce_backend' is undefined

  

    The error appears to have been in '/var/lib/jenkins/workspace/deepops_PR-302/playbooks/roce/mofed-install.yaml': line 17, column 7, but may

    be elsewhere in the file depending on the exact syntax problem.

  

    The offending line appears to be:

  

      tasks:

        - name: Check MLNX NIC's

          ^ here

	to retry, use: --limit @/var/lib/jenkins/workspace/deepops_PR-302/playbooks/k8s-cluster.retry
```

Reply:
Commented for future reason to select Ethernet, RoCE or Infiniband and CNI plugins.

Reply:
That's fine, but as-is the playbook will not run, so this variable needs to be defined by default in some way.

Reply:
Ok.  Will looking how to provide default parameters.